pkg/log/log_messages.go

```diff
@@ -75,3 +75,5 @@ const CAKC059 string = "CAKC059 Path exists but does not contain regular file: %
 const CAKC060 string = "CAKC060 Setting %s given invalid value %s"
 const CAKC061 string = "CAKC061 Failed to validate setting for Authenticator configuration"
 const CAKC062 string = "CAKC062 Required Authenticator setting %s not provided"
+const CAKC063 string = "CAKC063 Can't find configuration for URL: %s"
```

Comment:
In other repos we have been standardising on `Unable`. So these would all become `Unable to ...`